src/azure-cli/azure/cli/command_modules/containerapp/_utils.py

```diff
@@ -1342,10 +1342,10 @@ def queue_acr_build(cmd, registry_rg, registry_name, img_name, src_dir, dockerfi
         timeout=None,
         arguments=[])
 
-    queued_build = LongRunningOperation(cmd.cli_ctx)(client_registries.begin_schedule_run(
+    queued_build = client_registries.schedule_run(
         resource_group_name=registry_rg,
         registry_name=registry_name,
-        run_request=docker_build_request))
+        run_request=docker_build_request)
```

Comment:
+@Greedygre who is the owner of containerapp module. 
Could you please review this change as well?

Reply:
Hi @lizMSFT 
What is the difference between `begin_schedule_run` and `schedule_run`?
Do `schedule_run` include long running polling logic?